peribolos/knative.yaml

```diff
@@ -251,6 +251,7 @@ orgs:
     - joshlong
     - joshrider
     - joshua-bone
+    - joshuawilson
```

Comment:
Want to join knative-sandbox while you are at it?

Reply:
yes, please